sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2397,7 +2393,7 @@ export interface ListBlobsHierarchySegmentResponseModel {
 export type ListBlobsIncludeItem = "copy" | "deleted" | "metadata" | "snapshots" | "uncommittedblobs" | "versions" | "tags" | "immutabilitypolicy" | "legalhold" | "deletedwithversions";
 
 // @public
-export type ListContainersIncludeType = "metadata" | "deleted";
+export type ListContainersIncludeType = "metadata" | "deleted" | "system";
```

Comment:
It probably was a mistake to expose this type.  I don't think we use it anywhere in the public api surface.

Reply:
It's used in structure: `ContainerListBlobHierarchySegmentResponse ` which is return type for  `listBlobsByHierarchy ` method.

Reply:
My mistake, I thought you were talking about ListBlobsHierarchySegmentResponseModel ...

You are right, there's no interface using ListContainersIncludeType  or ListBlobsIncludeItem, they are only internally used. 

I've removed them.